src/game_api/rpc.cpp

```diff
@@ -499,11 +438,11 @@ Entity* get_entity_ptr(uint32_t uid)
 
 ENT_TYPE get_entity_type(uint32_t uid)
 {
-    auto state = State::get();
-    auto p = state.find(uid);
-    if (p == nullptr || IsBadWritePtr(p, 0x178))
-        return UINT32_MAX;
-    return p->type->id;
+    auto p = get_entity_ptr(uid);

```

Comment:
We don't like var names like `p`  ;)

Reply:
don't point at me, i didn't make it originally xd